source/browseMode.py

```diff
@@ -765,6 +817,15 @@ def script_disablePassThrough(self, gesture):
 	prevError=_("no previous error"))
 del qn
 
+PT = BrowseModeTreeInterceptor.addPassThrough

```

Comment:
Why don't you just add these to BrowseModeTreeInterceptor.__gestures?

Reply:
Just added. Joseph wanted me to write it this way.